code/lib/api/index.js

```diff
@@ -807,10 +807,64 @@ const api = {
     }
   },
   explorer: {
-    decodeMethod: (hash) => {
-      const { data } = axios.get(`https://explorer-v2-api.hmny.io/v0/signature/hash/${hash.slice(10)}`)
+    getApi: () => {
+      return activeNetwork === 'harmony-testnet' ? 'https://rpc.s0.b.hmny.io' : 'https://rpc.s0.t.hmny.io'
```

Comment:
need to check it is not local either. Also this should be in config / overridable by env

Reply:
Also we have a more dedicated RPC for wallet queries